finagle-redis/src/main/scala/com/twitter/finagle/redis/filter/RedisLoggingFilter.scala

```diff
@@ -1,12 +1,30 @@
 package com.twitter.finagle.redis.filter
 
-import com.twitter.finagle.{Service, SimpleFilter}
+import com.twitter.finagle._
 import com.twitter.finagle.redis.protocol._
 import com.twitter.finagle.redis.util.BufToString
 import com.twitter.finagle.stats.StatsReceiver
 import com.twitter.util.{Future, Return}
 
-private[redis] class RedisLoggingFilter(stats: StatsReceiver) extends SimpleFilter[Command, Reply] {
+private[finagle] object RedisLoggingFilter {
+  val role: Stack.Role = Stack.Role("RedisStatsFilter")
+  val description: String = "Redis Stats"
+
+  def module: Stackable[ServiceFactory[Command, Reply]] =
+    new Stack.Module1[param.Stats, ServiceFactory[Command, Reply]] {
+      val role: Stack.Role = RedisLoggingFilter.role
+      val description: String = RedisLoggingFilter.description
+
+      def make(statsParam: param.Stats, next: ServiceFactory[Command, Reply]): ServiceFactory[Command, Reply] = {
+        if (statsParam.statsReceiver.isNull)
+          next
+        else
+          new RedisLoggingFilter(statsParam.statsReceiver.scope("http")).andThen(next)
```

Comment:
I suspect the scope isn't really supposed to be `"http"`.

Reply:
Ah yes this looks copy-paste on my part, the original commit does not seem to add a scope. I can remove it. Alternatively some modules seem to scope the receiver to the protocol name like `http` so I could change it to that if it makes sense.